src/vega_viewer/vega/specs/stacked_horizontal_bar_chart.cljs

```diff
@@ -68,6 +72,17 @@
                                         :field "group"}}
                          :update {:fillOpacity {:value 1}}
                          :hover {:fillOpacity {:value 0.9}}}}
+           {:type "text"
+            :from {:data "grouped-data"}
+            :properties {:enter {:y {:scale "y"
+                                     :field "category"
+                                     :offset (-> bar-height
```

Comment:
It's how to center the bit of text vertically within the bar, in a way that will accommodate any possible future changes to the bar height. You're right, I'll pull them out.